nagios.go

```diff
@@ -0,0 +1,113 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os/exec"
+	"regexp"
+	"strings"
+)
+
+// NagiosData is a task factory for tasks that fetch a JSON representation of
+// the provided project. The task uses outFor and errOutFor to get io.Writers
+// to write, respectively, the JSON output and any eventual error message.
+func NagiosData(project string, outFor, errOutFor projectResourceWriterCloserFactory) Task {
+	return nagiosData(func(project, pod string) *exec.Cmd {
```

Comment:
@vchepeli I'm not sure what you mean here, if you are suggesting changing this to:
```
return nagiosData(func SomeFunctionNameHere(project, pod string) *exec.Cmd {
```
Then that is not valid syntax for go.

I can change it to:
```
        cmdFactory := func(project, pod string) *exec.Cmd {
		return exec.Command("oc", "exec", pod, "--", "cat", "/var/log/nagios/status.dat")
	}
	return nagiosData(cmdFactory, project, "Status", outFor, errOutFor)
```

If you find that syntax preferable, I will change it to that.